hbase-server/src/test/java/org/apache/hadoop/hbase/security/access/TestSnapshotScannerHDFSAclController.java

```diff
@@ -99,9 +98,11 @@ public static void setupBeforeClass() throws Exception {
           + SnapshotScannerHDFSAclController.class.getName());
 
     TEST_UTIL.startMiniCluster();
-    ProcedureTestingUtility.waitAllProcedures(
-      TEST_UTIL.getMiniHBaseCluster().getMaster().getMasterProcedureExecutor());
+    SnapshotScannerHDFSAclController coprocessor = TEST_UTIL.getHBaseCluster().getMaster()
+      .getMasterCoprocessorHost().findCoprocessor(SnapshotScannerHDFSAclController.class);
+    TEST_UTIL.waitFor(1200000, () -> coprocessor.checkInitialized("check initialized"));
```

Comment:
1_200_000 is too large I think... Could we make it smaller?

Reply:
Thank @Apache9 for reviewing.
> Mind explaining a bit on the reason why the test is flaky and how the PR here could fix it?

The coprocessor SnapshotScannerHDFSAclController will add 'm' family for hbase:acl after Master is started, `TEST_UTIL.waitTableAvailable(PermissionStorage.ACL_TABLE_NAME)` will scan all families of hbase:acl, so it is possible to scan 'm' family on regions that have not been modified and then get NoSuchColumnFamilyException. 
```
[ERROR] org.apache.hadoop.hbase.security.access.TestSnapshotScannerHDFSAclController  Time elapsed: 0.02 s  <<< ERROR!
java.lang.RuntimeException: 
org.apache.hadoop.hbase.regionserver.NoSuchColumnFamilyException: org.apache.hadoop.hbase.regionserver.NoSuchColumnFamilyException: Column family m does not exist in region hbase:acl,,1630063195592.47c0720458ae50f6692b73f906251e19. in table 'hbase:acl', {NAME => 'l', BLOOMFILTER => 'NONE', IN_MEMORY => 'true', VERSIONS => '1', KEEP_DELETED_CELLS => 'FALSE', DATA_BLOCK_ENCODING => 'NONE', COMPRESSION => 'NONE', TTL => 'FOREVER', MIN_VERSIONS => '0', BLOCKCACHE => 'true', BLOCKSIZE => '8192 B (8KB)', REPLICATION_SCOPE => '0'}
	at org.apache.hadoop.hbase.regionserver.HRegion.checkFamily(HRegion.java:5163)
	at org.apache.hadoop.hbase.regionserver.HRegion.lambda$getScanner$3(HRegion.java:3139)
	at org.apache.hadoop.hbase.trace.TraceUtil.trace(TraceUtil.java:219)
	at org.apache.hadoop.hbase.regionserver.HRegion.getScanner(HRegion.java:3128)
	at org.apache.hadoop.hbase.regionserver.HRegion.getScanner(HRegion.java:3123)
	at org.apache.hadoop.hbase.regionserver.HRegion.getScanner(HRegion.java:3117)
	at org.apache.hadoop.hbase.regionserver.RSRpcServices.newRegionScanner(RSRpcServices.java:3286)
	at org.apache.hadoop.hbase.regionserver.RSRpcServices.scan(RSRpcServices.java:3627)
	at org.apache.hadoop.hbase.shaded.protobuf.generated.ClientProtos$ClientService$2.callBlockingMethod(ClientProtos.java:45253)
	at org.apache.hadoop.hbase.ipc.RpcServer.call(RpcServer.java:396)
	at org.apache.hadoop.hbase.ipc.CallRunner.run(CallRunner.java:136)
	at org.apache.hadoop.hbase.ipc.RpcExecutor$Handler.run(RpcExecutor.java:338)
	at org.apache.hadoop.hbase.ipc.RpcExecutor$Handler.run(RpcExecutor.java:318)

	at org.apache.hadoop.hbase.security.access.TestSnapshotScannerHDFSAclController.setupBeforeClass(TestSnapshotScannerHDFSAclController.java:104)
```
So what this PR does is that waiting for the coprocessor initialization(table modification) to complete after the cluster is started, and then `waitTableAvailable` for hbase:acl.

> 1_200_000 is too large I think... Could we make it smaller?

Made it smaller to 30000.